src/main/java/gregtech/api/capability/impl/AbstractRecipeLogic.java

```diff
@@ -414,6 +477,12 @@ public void deserializeNBT(NBTTagCompound compound) {
         if(compound.hasKey("AllowOverclocking")) {
             this.allowOverclocking = compound.getBoolean("AllowOverclocking");
         }
+        if (compound.hasKey("OverclockVoltage")) {
```

Comment:
Can you make this constants so we don't have to have 3 literals with same text. And if possible do same for `AllowOverclocking` (I know you did not added but when you are working on it...)